### DISCUSSION TITLE
New Quire publication: "Heaven & Earth: The Blue Maps of China," Leventhal Map & Education Center at the Boston Public Library, 2024

### DISCUSSION THREAD

**Discussion by garrettdashnelson** [thegetty/quire#931] [Show & Tell]
Production site: https://www.leventhalmap.org/digital-exhibitions/heaven-and-earth/

Source repo: https://github.com/bplmaps/heaven-and-earth

**Comment by Erin-Cecele** [comment]
This is very exciting @garrettdashnelson. Wonderful to see another publication from the Leventhal Map & Education Center and on that you smoothly transitioned to v1. Thank you so much for sharing this with the community!
